lib/msfl/sinatra.rb

```diff
@@ -0,0 +1,90 @@
+module MSFL
+  module Sinatra
+    class << self
+
+      attr_accessor :valid_filter
+
+      def valid_filter
+        @valid_filter ||= nil
+      end
+
+      # Extracts the filter clause from a Sinatra request params hash and parsers the filter
+      #
+      # @param params [Hash] the Sinatra request params
+      # @return [Object] the Ruby-ified MSFL filter
+      def parse_filter_from(params)
```

Comment:
the _from suffix naming isn't clicking for me, what does it mean? _from_params(params)?